aspnetcore/fundamentals/openapi/samples/9.x/BuildTime/Program.cs

```diff
@@ -0,0 +1,46 @@
+using System.Reflection;
+
+var builder = WebApplication.CreateBuilder();
+
+if (Assembly.GetEntryAssembly()?.GetName().Name != "GetDocument.Insider")
+{
+   // builder.Services.AddDefaults();
+}
```

Comment:
`builder.Services.AddDefaults();` generates the error:
'IServiceCollection' does not contain a definition for 'AddDefaults'

Reply:
This is my bad...I had trouble figuring out what would be a good illustration of a setup here.

Maybe we can do something that configures an EF Core DB context?

```
services.AddDbContext<MyContext>(options =>
                options.UseSqlServer(Configuration.GetConnectionString("DefaultConnection")));
```

We'd have to add a definition for `MyContext` and the package references for EF

Reply:
@captainsafia @martincostello 

> This is my bad...I had trouble figuring out what would be a good illustration of a setup here.
> 
> Maybe we can do something that configures an EF Core DB context?
> 
> ```
> services.AddDbContext<MyContext>(options =>
>                 options.UseSqlServer(Configuration.GetConnectionString("DefaultConnection")));
> ```
> 
> We'd have to add a definition for `MyContext` and the package references for EF

I have:
* [Context](https://github.com/Rick-Anderson/BuildTimeEFconStr/blob/main/Program.cs#L6-L11) 
* [package references for EF](https://github.com/Rick-Anderson/BuildTimeEFconStr/blob/main/BuildTimeEFconStr.csproj#L11)
* [Connection string](https://github.com/Rick-Anderson/BuildTimeEFconStr/blob/main/appsettings.json#L10)